src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -497,6 +497,10 @@ def _get_identity_object_from_type(
 
                 identityResult = ResourceIdentity(type=ResourceIdType.user_assigned.value,
                                                   user_assigned_identities=umiDict)
+
+            if resourceIdentityType == ResourceIdType.system_assigned.value:
+                identityResult = ResourceIdentity(type=ResourceIdType.system_assigned.value)
```

Comment:
```suggestion
        if resourceIdentityType == ResourceIdType.system_assigned.value:
            identityResult = ResourceIdentity(type=ResourceIdType.system_assigned.value)
```
Do I read it wrong or is it really inside the if clause for `resourceIdentityType == ResourceIdType.user_assigned.value`?